packages/react/src/search-box/docs/overview.mdx

```diff
@@ -1,6 +1,6 @@
 ---
-title: Search Box
-description: Search input enables users to enter a word or phrase to find relevant content.
+title: Search box
+description: An input that allows users to enter a keyword to search a website for specific content.
```

Comment:
While I was here I noticed this overview is wrong. I've updated this content to what's in the Figma gallery.